cmd/syncthing/gui.go

```diff
@@ -691,6 +697,17 @@ func (s *apiService) getSystemStatus(w http.ResponseWriter, r *http.Request) {
 	res["uptime"] = int(time.Since(startTime).Seconds())
 	res["startTime"] = startTime
 
+	seen := make(map[string]struct{})
+	var themes []string
+	for file := range auto.Assets() {
```

Comment:
Since this only looks at the compiled in assets, the work should be done at asset build time. But maybe it should also look at files in the STGUIASSETS dir? A listdir there or something. Although nothing too expensive, as this gets called every few seconds by the GUI (and other stuff that don't care about the theme, e.g. synctrayzor and such)...